homeassistant/components/roon/media_player.py

```diff
@@ -288,6 +301,16 @@ def update_state(self):
         self._attr_is_volume_muted = volume["muted"]
         self._attr_volume_step = volume["step"]
         self._attr_volume_level = volume["level"]
+        self.volume_fixed = volume["fixed"]
+        self.volume_incremental = volume["incremental"]
```

Comment:
How does this work?
`volume["fixed"]` and `volume["incremental"]` is only set in the `else` clause of the try-except above, is it OK to access them unconditionally here?